factory/glideFactoryMonitorAggregator.py

```diff
@@ -913,15 +827,18 @@ def aggregateRRDStats(log=logSupport.log):
         # assigns the data from every site to 'stats'
         stats = {}
         for entry in monitorAggregatorConfig.entries:
-            rrd_fname = os.path.join(os.path.join(monitorAggregatorConfig.monitor_dir, "entry_" + entry), rrd_site(rrd))
+            rrd_fname = os.path.join(monitorAggregatorConfig.monitor_dir, "entry_" + entry, rrd_site(rrd))
```

Comment:
Would it be better to change `"entry_" + entry` to `f"entry_{entry}"`?

Reply:
done